src/client/vite.config.ts

```diff
@@ -219,32 +219,43 @@ const injectScriptIntoHtml = (
   env: string,
 ) =>
   createHtmlPlugin({
-    inject: {
-      data: {
-        injectScript: `
-          ${
-            buildTarget === "web"
-              ? `<link rel="manifest" href="${getBaseUrl(
-                  isDev,
-                )}/manifest.json" />`
-              : "<!-- no manifest.json for OpenFin -->"
-          }
-          
-          <script>
-            // Hydra dependency references BigInt at run time even when the application isn't explicitly started
-            // Detect this as supportsBigInt so we  can show a 'browser unsupported' message
-            // Set BigInt to an anon function to prevent the runtime error
-
-            window.supportsBigInt = typeof BigInt !== 'undefined';
-            window.BigInt = supportsBigInt ? BigInt : function(){};
-          </script>
-          
-          <script async src="https://www.googletagmanager.com/gtag/js?id=${
-            env === "prod" ? "G-Z3PC9MRCH9" : "G-Y28QSEPEC8"
-          }"></script>
-        `,
+    pages: [
```

Comment:
well, this was a right palaver .. just to get the ["multipage app" config](https://vitejs.dev/guide/build.html#multi-page-app) going was sweat and a few tears